### ISSUE TITLE
Reflexmate in 1 - intended behavior or broken?

### ISSUE THREAD

**Issue by dturevski** [thomas-maeder/popeye#313]
```
BeginProblem
Stipulation r#1
Pieces white  Ke6 Ra1 black  Ke8
EndProblem

Popeye Windows-64Bit v4.85 (4000 MB)
   1.Ra1-a8 #
solution finished. Time = 0.001 s
```

This is the only legal move and it checkmates the black king. However, how does it fulfill the stipulation that requires that the white king is checkmated?  Changing the stipulation to semi-r#1 produces no solution, as expected.

**Comment by dturevski** [comment]
I think I kind of know what happens. There's no ! mark after the move, so it's not a key, but a sort of refutation. But the input did not ask for variations or tries, so this is not really consistent with how the other direct stipulations work, they only list the keys.

**Comment by thomas-maeder** [comment]
I added this kind of output in 4.53 to help understand why White can't execute other moves.

I think that you have a point, but I'm not sure what to do about it.
The move is neither a try nor a refutation nor a variation.

**Comment by dturevski** [comment]
> I'm not sure what to do about it.
- Document it
- Remove it
- Both of the above: make it an user option

**Comment by JoshuaGreen** [comment]
I'll offer a fourth option:
- Extend it -- whenever _any_ problem has a forced initial sequence of moves, provide it with an appropriate label.

**Comment by thomas-maeder** [comment]
Implemented for 4.89
